website/pages/en/developing/creating-a-subgraph/install-the-cli.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: Install the Graph CLI
+---
+
+> In order to use your subgraph on The Graph's decentralized network, you will need to [create an API key](/deploying/subgraph-studio-faqs/#2-how-do-i-create-an-api-key) in [Subgraph Studio](https://thegraph.com/studio/apikeys/). It is recommended that you add signal to your subgraph with at least 3,000 GRT to attract 2-3 Indexers. To learn more about signaling, check out [curating](/network/curating/).
+
+## Overview
+
+The [Graph CLI](https://github.com/graphprotocol/graph-tooling/tree/main/packages/cli) is a command-line interface that facilitates developers' commands for The Graph. It processes a [subgraph manifest](/creating-a-subgraph/subgraph-manifest/) and compiles the [mappings](/creating-a-subgraph/assemblyscript-mappings/) to create a ready-to-use version of the subgraph.
+
+You need The Graph CLI to build and deploy a subgraph.
+
+## Getting Started
+
+### Install the Graph CLI
+
+The Graph CLI is written in TypeScript, and you must have `node` and either `npm` or `yarn` installed to use it. Check for the [most recent](https://github.com/graphprotocol/graph-tooling/releases?q=%40graphprotocol%2Fgraph-cli&expanded=true) CLI version.
+
+On your local machine, run one of the following commands:
+
+#### Using [npm](https://www.npmjs.com/)
+
+```bash
+npm install -g @graphprotocol/graph-cli@latest
+```
+
+#### Using [yarn](https://yarnpkg.com/)
+
+```bash
+yarn global add @graphprotocol/graph-cli
+```
+
+- The `graph init` command can be used to set up a new subgraph project, either from an existing contract or from an example subgraph.
+
+- This `graph init` command can also create a subgraph in Subgraph Studio by passing in `--product subgraph-studio`.
+
+- If you already have a smart contract deployed to your preferred network, you can bootstrap a new subgraph from that contract to get started.
```

Comment:
Instead of a bulleted list, I would just make this a paragraph. I would just say that users can create a new subgraph from an existing contract or an example subgraph, and the the following subsections explain the details. I am not sure what the `--product subgraph-studio` flag is trying to tell me here, and it raises the question of what happens when I don't include that.

Not sure who's best equipped to answer that, either studio folks or Graph CLI devs?